test/test_datasets.py

```diff
@@ -383,12 +387,84 @@ def test_places365_images_download_preexisting(self):
                 torchvision.datasets.Places365(root, split=split, small=small, download=True)
 
     def test_places365_repr_smoke(self):
-        with places365_root(extract_images=False) as places365:
+        with places365_root() as places365:
             root, data = places365
 
             dataset = torchvision.datasets.Places365(root, download=True)
             self.assertIsInstance(repr(dataset), str)
 
 
+class STL10Tester(DatasetTestcase):
```

Comment:
After adding the tests for `Places365` we agreed on creating a separate test case for a dataset if more than the generics are checked